lms/djangoapps/courseware/masquerade.py

```diff
@@ -91,7 +98,7 @@ def get(self, request, course_key_string):
             user_name=None,
         )
         descriptor = modulestore().get_course(course_key)
-        partitions = get_all_partitions_for_course(descriptor)
+        partitions = get_all_partitions_for_course(descriptor, active_only=True)
```

Comment:
I didn't realize the function took this optional kwarg;
we don't want to list inactive masquerade tracks.

Note: This is only consumed by the MFE now.